projects/dsp-ui/src/lib/action/pipes/formatting/knoradate.pipe.ts

```diff
@@ -29,4 +46,16 @@ export class KnoraDatePipe implements PipeTransform {
         return ('0' + value).slice(-2);
     }
 
+    // add the era, calendar, or both to the result returned by the pipe
+    addPrecision(date: KnoraDate, value: string, precision: string): string {
```

Comment:
see my comment above about naming it differently

Reply:
fixed in b40ef39